docs/index.asciidoc

```diff
@@ -20,40 +20,52 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
-This filter is best suited for event enhancement from static or
-reference data that does not change very often. Examples of these
-are environments, users and products. Locally there is a database
-that the enhancement lookups are done against. Remotely there is
-a database that has the static data. The local db is loaded with
-data from the remote db once or very periodically. To achieve this
-three main sections are defined: db_objects, loaders and lookups.
-The local database is an
-https://db.apache.org/derby/manuals/#docs_10.14[Apache Derby]
-in-memory database.
+This filter enriches events with data pre-loaded from a remote database. 
 
-Db_objects
+This filter is best suited for enriching events with reference data that is
+static or does not change very often, such as environments, users, and products.
 
-Define as many of theses as needed to build the local database
-structure. Define tables, columns and indexes.
+This filter works by fetching data from a remote database, caching it in a
+local, in-memory https://db.apache.org/derby/manuals/#docs_10.14[Apache Derby]
+database, and using lookups to enrich events with data cached in the local
+database. You can set up the filter to load the remote data once (for static
+data), or you can schedule remote loading to run periodically (for data that
+needs to be refreshed).
 
-Loaders
+//REVIEWERS: It's true that you can just load once, right, if you don't specify
```

Comment:
that is my understanding as well. @guyboertje can confirm.

Reply:
Yes once - unless a schedule is specified (that actually works!!).